File: Atividade Avaliativa Final/Javascript/entity/torneio/TorneioBasquete.js
import { Torneio } from './Torneio.js';

export class TorneioBasquete extends Torneio {
  constructor(nome) {
    super(nome);
  }

  getModalidade() {
    return 'Basquete';
  }

  getPosicoes() {
    return ['Armador', 'Ala', 'Pivô'];
  }
}

File: Atividade Avaliativa Final/Python/entity/torneio/torneio_basquete.py


from entity.torneio.torneio import Torneio

class TorneioBasquete(Torneio):
    def get_modalidade(self) -> str:
        return "Basquete"

    def get_posicoes(self) -> list:
        return ["Armador", "Ala-armador", "Ala", "Ala-pivô", "Pivô"]